src/main/java/graphql/execution/ExecutionStrategy.java

```diff
@@ -692,7 +692,7 @@ private Object handleCoercionProblem(ExecutionContext context, ExecutionStrategy
     }
 
     protected GraphQLObjectType resolveType(ExecutionContext executionContext, ExecutionStrategyParameters parameters, GraphQLType fieldType) {
-        return resolvedType.resolveType(executionContext, parameters.getField(), parameters.getSource(), parameters.getExecutionStepInfo(), fieldType, parameters.getLocalContext());
+        return resolvedType.resolveType(executionContext, parameters.getField(), parameters.getSource(), parameters.getExecutionStepInfo().getArguments(), fieldType);
```

Comment:
why these changes?  What has this got to do with instrumentation state?

Reply:
same as above